engine/src/main/java/org/terasology/rendering/world/WorldRendererLwjgl.java

```diff
@@ -197,101 +266,97 @@ public void render(DefaultRenderingProcess.StereoRenderState stereoRenderState)
         switch (stereoRenderState) {
             case MONO:
                 currentRenderingStage = WorldRenderingStage.DEFAULT;
+                isFirstRenderingStageForCurrentFrame = true;
                 break;
             case OCULUS_LEFT_EYE:
                 currentRenderingStage = WorldRenderingStage.OCULUS_LEFT_EYE;
                 playerCamera.updateFrustum();
+                isFirstRenderingStageForCurrentFrame = true;
                 break;
             case OCULUS_RIGHT_EYE:
                 currentRenderingStage = WorldRenderingStage.OCULUS_RIGHT_EYE;
                 playerCamera.updateFrustum();
+                isFirstRenderingStageForCurrentFrame = false;
                 break;
         }
 
-        if (stereoRenderState == DefaultRenderingProcess.StereoRenderState.MONO || stereoRenderState == DefaultRenderingProcess.StereoRenderState.OCULUS_LEFT_EYE) {
```

Comment:
It might be nice if all the first-rendering-stage stuff could be placed in a method and just called first off, before rendering each "eye". Don't know how interwoven it all is.